app/main.py

```diff
@@ -104,6 +105,19 @@ def fetch_vocabularies(config_name: str) -> dict:
             )
             all_std_trm_vocabs[var_uri] = std_trm_vocab
 
+    # Imaging modalities vocab is stored alongside config.json, not listed as a standardized variable.
+    # therefore we fetch the imaging modalities vocab from the Neurobagel directory separately.
+    imaging_vocab_uri = f"{std_var_config['namespace_prefix']}:Image"
+    imaging_vocab_config_dir_url = util.create_gh_raw_content_url(
+        env_settings.NEUROBAGEL_CONFIG_REPO,
+        "configs/Neurobagel/imaging_modalities.json",
+    )
```

Comment:
```suggestion
    imaging_vocab_url = util.create_gh_raw_content_url(
        env_settings.NEUROBAGEL_CONFIG_REPO,
        "configs/Neurobagel/imaging_modalities.json",
    )
```
since it's pointing directly to the vocab file and not to a config directory